src/fragments/start/getting-started/flutter/integrate.mdx

```diff
@@ -228,230 +202,370 @@ class _AddTodoFormState extends State<AddTodoForm> {
 }
 ```
 
-Go ahead and run your code now on an iOS or Android device and you should see an app with a progress indicator and a floating action button but not much else.
+Go ahead and run your code now on any device and you should see an app with some titles and a floating action button but not much else.
```

Comment:
Go ahead and run your code now on any mobile, web, or desktop device and you should see an app with some titles and a floating action button but not much else.